### DISCUSSION TITLE
Quizzes report FP but do not report FN when multiple answers are valid

### DISCUSSION THREAD

**Discussion by agaldran** [Lightning-AI/dl-fundamentals#17] [Lecture videos and quizzes]
Just as the title shows, if a valid answer is not marked, this error will not be reported and we don't know we got it wrong, see below:

![image](https://user-images.githubusercontent.com/4498444/212799570-3349f839-5371-463b-8da4-ea010e6da0a6.png)

Above, the first option was also a valid answer, I did not mark it, but this was not repored as incorrect.

Thanks for the materials!

**Comment by rasbt** [comment]
Sorry, I totally missed that, and thanks for the note! We will be fixing/improving that!
